### DISCUSSION TITLE
CI and previews from PRs

### DISCUSSION THREAD

**Discussion by lazarusA** [vuejs/vitepress#3415] [Q&A]
is it possible activate/deploy previews from PRs? so that we can see how things will look like before merging into main.

**Comment by brc-dd** [comment]
You mean in this repo? They are already enabled. You need to click details at the bottom or the commit checks.

**Reply by lazarusA** [reply]
Ohh.. yes I see... thanks. 

I was looking more for something like this: https://github.com/zhangyx1998/vitepress/blob/feat/multithread-render/.github/workflows/test.yml
